packages/cozy-urls/src/cozy-urls.js

```diff
@@ -6,11 +6,7 @@ const ERROR_DOMAIN_NODE = `[cozy-url] COZY_URL variable isn't defined.`
 const ERROR_PROTOCOL_URL_INVALID = `[cozy-urls] Can't find protocol for`
 const ERROR_DOMAIN_URL_INVALID = `[cozy-urls] Can't find domain for`
 
-let cozyURL
-
 export const getBrowserCozyURL = () => {
-  if (cozyURL) return cozyURL
```

Comment:
Maybe we can use localstorage here? Without that, each time we will want to get the CozyURL it will do a DOM request. Or maybe we have to explain that detail in the readme to not use it everywhere in the application code.

Reply:
We will talk about optimization when the library will exist